cmd/nfd-topology-updater/main.go

```diff
@@ -81,7 +81,7 @@ func parseArgs(flags *flag.FlagSet, osArgs ...string) (*topology.Args, *resource
 
 	_ = flags.Parse(osArgs)
 	if len(flags.Args()) > 0 {
-		fmt.Fprintf(flags.Output(), "unknown command line argument: %s\n", flags.Args()[0])
+		_, _ = fmt.Fprintf(flags.Output(), "unknown command line argument: %s\n", flags.Args()[0])
```

Comment:
nit: I think this is unnecessary 

```suggestion
		fmt.Fprintf(flags.Output(), "unknown command line argument: %s\n", flags.Args()[0])
```